tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/TestContexts/OdsRepositoryTestContext.cs

```diff
@@ -13,7 +13,9 @@ private OdsRepositoryTestContext()
                 BuyerOrganisationRoleIds = new[] { "RO98", "RO177", "RO213", "RO272" },
             };
 
-            OdsRepository = new OdsRepository(OdsSettings);
+            LazyCache.CachingService.DefaultCacheProvider.Value.Remove($"Ods-Org-XYZ");
```

Comment:
Looks like a redundant `$` here as we're not doing any interpolation.

Reply:
removed